jetty-ee10/jetty-ee10-jaas/src/main/java/org/eclipse/jetty/ee10/jaas/spi/PropertyFileLoginModule.java

```diff
@@ -97,7 +99,9 @@ private void setupPropertyUserStore(Map<String, ?> options)
                 }
             }
             PropertyUserStore newStore = new PropertyUserStore();
-            newStore.setConfig(filename);
+            ResourceFactory resourceFactory = ResourceFactory.of(newStore);
+            Resource config = resourceFactory.newResource(filename);
+            newStore.setConfig(config);
```

Comment:
Shouldn't `PropertyUserStore.setConfig()` still accept a string and do that logic internally?

Reply:
The config that winds up here is also used by other things like HashLoginService.

The change to support String with internal ResourceFactory was overly complicated when I started to look at it (was caught up in a chain of LifeCycle's that didn't seem to operate normally).

But it's a good TODO for a later PR, which I'll add here.